### ISSUE TITLE
Rename "Clear Shared Data" option in settings. 

### ISSUE THREAD

**Issue by Luke-Ed** [killpowa/Twickt-Launcher#1336]
This option's name doesn't clearly represent what it does to the user. From the developer point of view, it does, however when reading it from a user perspective, I assumed it would be clearing a cache of shared mods. I deleted all of my instances, and worlds by choosing this option. 

While this is user error, an option like this should have a more defined name / description, using a different color/underlining to make it clearer to the user that this will be deleting all of their instances, and their associated data.

**Comment by oitsjustjose** [comment]
Agreed - also a dev (even worked on this launcher) and had this issue a few months back. I didn't have any backup because I never expected an application to clear out that data.

**Comment by huantianad** [comment]
Perhaps it would be best to remove the option entirely, and instead have the user manually delete the folder.
